canonical-websites/www.ubuntu.com
# Add short product urls

## Done

## Existing

- [bootstack](http://0.0.0.0:8001/bootstack)             ->          /cloud/managed-cloud
- [core](http://0.0.0.0:8001/core)                      ->          /core
- [juju](http://0.0.0.0:8001/juju)                        ->          /cloud/juju
- [kubernetes](http://0.0.0.0:8001/kubernetes)           ->          /kubernetes
- [landscape](http://0.0.0.0:8001/landscape)             ->          https://landscape.canonical.com/
- [livepatch](http://0.0.0.0:8001/livepatch)               ->          https://auth.livepatch.canonical.com/
- [lxd](http://0.0.0.0:8001/lxd)                         ->          /containers/lxd
- [maas](http://0.0.0.0:8001/maas)                     ->          /server/maas
- [openstack](http://0.0.0.0:8001/openstack)             ->          /openstack
- [snappy](http://0.0.0.0:8001/snappy)                  ->          /desktop/snappy

## Added

- [docker-engine](http://0.0.0.0:8001/docker-engine)                        ->          /containers/docker-ubuntu
- [foundation-cloud-build](http://0.0.0.0:8001/foundation-cloud-build)          ->          /cloud/foundation-cloud
- [foundation-cloud](http://0.0.0.0:8001/foundation-cloud)                    ->          /cloud/foundation-cloud
- [ubuntu-advantage](http://0.0.0.0:8001/ubuntu-advantage)                  ->          /support
- [ubuntu-advantage-storage](http://0.0.0.0:8001/ubuntu-advantage-storage)    ->          /cloud/storage


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check the 'Added' list to make sure the redirects work


## Issue / Card

Brief: https://docs.google.com/document/d/1RNBnu8qOetkXBUt1LdVClgCDwM6aCUwd636YqZO5mis/edit#
Fixes: https://app.zenhub.com/workspace/o/ubuntudesign/web-squad/issues/238
